doc/source/cluster/kubernetes/k8s-ecosystem/yunikorn.md

```diff
@@ -0,0 +1,190 @@
+
+First, create a queue with a capacity of 4 CPUs and 6Gi of RAM by editing the ConfigMap:
+
+Run `kubectl edit configmap -n yunikorn yunikorn-defaults`
+
+This `ConfigMap` is created during the installation of the Apache YuniKorn Helm chart.
+
+Add a `queues.yaml` config to under the `data` key, the final ConfigMap should look like this:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  # Metadata for the ConfigMap, skip for brevity
+data:
+  queues.yaml: |
+    partitions:
+      - name: default
+        queues:
+          - name: root
+            queues:
+              - name: test
+                submitacl: "*"
+                parent: false
+                resources:
+                  guaranteed:
+                    memory: 6G
+                    vcore: 4
+                  max:
+                    memory: 6G
+                    vcore: 4
+```
+
+Save the changes and exit the editor. This configuration creates a queue named `root.test` with a capacity of 4 CPUs and 6Gi of RAM.
+
+Next, create a RayCluster with a head node (1 CPU + 2Gi of RAM) and two workers (1 CPU + 1Gi of RAM each), for a total of 3 CPU and 4Gi of RAM:
+
+```shell
+# Path: kuberay/ray-operator/config/samples
+# Configure the necessary labels on the RayCluster custom resource for Apache YuniKorn scheduler's gang scheduling:
+# - `ray.io/gang-scheduling-enabled`: This should be set to `true` to enable gang scheduling.
+# - `yunikorn.apache.org/app-id`: This should be set to a unique identifier for the application in Kubernetes, even across different namespaces.
+# - `yunikorn.apache.org/queue`: This should be set to the name of one of the queues in Apache YuniKorn.
+wget https://raw.githubusercontent.com/ray-project/kuberay/master/ray-operator/config/samples/ray-cluster.yunikorn-scheduler.yaml
+kubectl apply -f ray-cluster.yunikorn-scheduler.yaml
+```
+
+Check the RayCluster that was created:
+
+```shell
+$ kubectl describe raycluster test-yunikorn-0
+
+Name:         test-yunikorn-0
+Namespace:    default
+Labels:       ray.io/gang-scheduling-enabled=true
+              yunikorn.apache.org/app-id=test-yunikorn-0
+              yunikorn.apache.org/queue=root.test
+Annotations:  <none>
+API Version:  ray.io/v1
+Kind:         RayCluster
+Metadata:
+  Creation Timestamp:  2024-09-29T09:52:30Z
+  Generation:          1
+  Resource Version:    951
+  UID:                 cae1dbc9-5a67-4b43-b0d9-be595f21ab85
+# Other fields are skipped for brevity
+````
+
+Note the labels on the RayCluster: `ray.io/gang-scheduling-enabled=true`, `yunikorn.apache.org/app-id=test-yunikorn-0`, and `yunikorn.apache.org/queue=root.test`.
+
+:::{note}
+
+The `ray.io/gang-scheduling-enabled` label is only needed when Gang Scheduling is required. If this label is not set, the Ray cluster will be scheduled by YuniKorn without enforcing Gang Scheduling.
+
+:::
+
+Because the queue has a capacity of 4 CPU and 6Gi of RAM, this resource should schedule successfully without any issues.
```

Comment:
```suggestion
Because the queue has a capacity of 4 CPU and 6GB of RAM, this resource should schedule successfully without any issues.
```

Reply:
GB is different from GiB. I've changed it to GiB